tests/Finder/TestFrameworkFinderTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Tests\Finder;
+
+use Infection\Finder\Exception\FinderException;
+use Infection\Finder\TestFrameworkFinder;
+use PHPUnit\Framework\TestCase;
+
+class TestFrameworkFinderTest extends TestCase
+{
+    public function test_it_can_load_a_custom_path()
+    {
+        $filename = '/tmp/infection-test-framework-finder-test' . uniqid('', false);
```

Comment:
this test won't work on windows I think. Could you please use `sys_get_temp_dir` instead? also it'll be a good idea to clean up.

you can use symfony filesystem class for example. We already have couple tests which do similar things.